app/models.py

```diff
@@ -633,9 +623,6 @@ def from_json(cls, data):
         # validate json with marshmallow
         fields = data.copy()
 
-        # TODO: remove this line once admin stops passing through normalised_service_name:
-        fields.pop("normalised_service_name", None)
```

Comment:
Have we cleared all the service caches since this happened? I don't think anything passes it through even if it's in the cache, but just in case.

Reply:
yup, i cleared all the caches at the end of last wk 👍